### DISCUSSION TITLE
Rounded corners for notification icon

### DISCUSSION THREAD

**Discussion by julius-boettger** [ErikReider/SwayNotificationCenter#348] [Q&A]
Hello,
I just can't figure out how to get rounded corners on the notification icons. It currently looks like this, but I would much prefer it to have rounded corners.
![image](https://github.com/ErikReider/SwayNotificationCenter/assets/85450899/fecd0c41-ab85-4830-a4a1-604efe70af02)
Any help is appreciated!

**Comment by ErikReider** [comment]
Moving into a issue: #349
